inc/fiber_prompt.h

```diff
@@ -0,0 +1,46 @@
+/** A basic fiber interface. **/
+#ifndef WASMFX_FIBER_C_H
+#define WASMFX_FIBER_C_H
+
+#define export(NAME) __attribute__((export_name(NAME)))
+
+/** The abstract type of a prompt object **/
+typedef struct prompt* prompt_t;
```

Comment:
```suggestion
typedef uint32_t prompt_t;
```
I suggest making it concrete.